crates/core/tedge_agent/src/entity_manager/tests.rs

```diff
@@ -396,6 +397,7 @@ mod entity {
         let server = EntityStoreServer::new(
             entity_store,
             mqtt_schema,
+            MqttConfig::default(),
```

Comment:
To avoid any dependency on test-mqtt-server from this unit test code, a different abstraction is needed here so that dummy messages can be passed to the test code without using a broker..